numpy/typing/_callable.py

```diff
@@ -103,6 +112,15 @@ def __call__(self, __other: float) -> floating: ...
         @overload
         def __call__(self, __other: complex) -> complexfloating[floating]: ...
 
+    class _UnsignedIntBitOp(Protocol):
+        # TODO: The likes of `uint64 | np.signedinteger` will fail as there
+        # is no signed integer type large enough to hold a `uint64`
+        # See https://github.com/numpy/numpy/issues/2524
```

Comment:
This comment seems misplaced to me, it applies to the second overload only.

Reply:
This is correct. Inserting a comment between two overloads looks visually rather bad though, hence why it is on top.